### DISCUSSION TITLE
Build and Run error

### DISCUSSION THREAD

**Discussion by AyushMathpal** [Atri-Labs/atrilabs-engine#660] [Help: Miscellaneous]
The application is working on localhost fine but when I try to run and build, it shows an error like this.
![image](https://user-images.githubusercontent.com/92756482/217586689-f10a4a10-e2f7-45ac-a70f-d6570efa1d2b.png)

The error says=>"Some error has occurred. We are very sorry for this. Please contact the Atri Labs team at their Slack channel for help."
The terminal is clean and I have tried both Brave and Chrome. Please Help

**Comment by darshitac11** [comment]
Would you mind trying `Build and Run` again?

I cloned the project and followed all the steps [here](https://github.com/Atri-Apps/restaurant_website#how-to-use-this-app) but I didn't get any error.

**Reply by AyushMathpal** [reply]
I didn't clone the restaurant project to be precise. I am making it by myself in my local system. That's when I got the error.

**Reply by darshitac11** [reply]
Ah okay. 

In that case, stop the terminal, run `atri start --debug` and try `Build and Run` again. Let us know if you still get this error.

**Reply by AyushMathpal** [reply]
![image](https://user-images.githubusercontent.com/92756482/217759434-3113a993-7831-4c18-97ef-cbbe4df2f1d4.png)
This is the error that pops up after following the aforementioned steps. I tried resolving it but not able to understand the flow of files. Can you help me resolve this error?

**Reply by cruxcode** [reply]
@AyushMathpal sorry you facing this problem. We will fix it in a few days. We are working on a major revamp, that will take care of all these issues. Please wait for a few days. You can keep working on your project in the visual editor and you don't need to wait. We will make sure that you are able to migrate to the new version we are about to release very easily.

**Comment by Shubhodeep100** [comment]
![Screenshot 2023-02-13 232526](https://user-images.githubusercontent.com/96099026/218634623-fa899603-6a27-4584-b650-83dc055abd59.jpg)
On running the 'Build and run' button on Atri Editor, localhost:4005 is available to open. But I'm getting this error message on the localhost:4005. Please help me resolve this issue.

**Reply by Shubhodeep100** [reply]
@cruxcode @darshitac11 Please help fix this error message.
